mp/src/creategameprojects.ps1

```diff
@@ -8,7 +8,7 @@ if (-NOT ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdent
 $ErrorActionPreference= 'silentlycontinue'
 # This is the path to your steamworks dev folder (depo). This should be the folder with hl2.exe inside of it.
 # Your local momentum folder will be sym linked INSIDE this folder as momentum
-$path = "E:\Steamworks\dev\Momentum"
+$path = "C:\Users\eelih\Documents\Koodit\Momentum\MomentumDev"
```

Comment:
Don't commit this change, you can revert this file back after creating the symlink since it only needs to be run once.